### PR TITLE
Update config.js

### PR DESCRIPTION
**Pull request by Heus-Sueh**
adding lang, cleanUrls, ignoreDeadLinks, docFooter and related settings to the mobile page

### CODE REVIEW

**Comment by Heus-Sueh** [comment]
added **editLink**

File: docs/.vitepress/config.js [hunk 1]
```diff
@@ -72,6 +72,10 @@ export default {
       prev: false,
       next: true,
     },
+    editLink: {
```
Comment (Evavic44) [inline docs/.vitepress/config.js:75]: Edit link not working. Should follow [docs format](https://vitepress.dev/reference/default-theme-config#editlink)
```js
https://github.com/Evavic44/adocs/edit/main/docs/:path
```

**Review by Evavic44 — CHANGES_REQUESTED** [review]
Thanks for the PR. Please fix the issues so I can merge it.

**Comment by Heus-Sueh** [comment]
> Thanks for the PR. Please fix the issues so I can merge it.

Okay, I hadn't seen the error before

**Pull request merged**